mainframer.sh

```diff
@@ -12,8 +12,9 @@ PROJECT_DIR=$DIR
 PROJECT_DIR_NAME="$( basename "$PROJECT_DIR")"
 MAINFRAMER_DIR="$PROJECT_DIR/.mainframer"
 PERSONAL_CONFIG_FILE="$MAINFRAMER_DIR/personalconfig"
-IGNORE_LOCAL_FILE="$MAINFRAMER_DIR/localignore"
-IGNORE_REMOTE_FILE="$MAINFRAMER_DIR/remoteignore"
+LOCAL_IGNORE_FILE="$MAINFRAMER_DIR/localignore"
```

Comment:
Sorry, renamed these variables to be consistent with `tests/common.sh` (new variant better imo)